src/app/modules/OpenshiftStreams/components/StreamsTable/StreamsTable.tsx

```diff
@@ -172,7 +172,7 @@ export const StreamsTable: FunctionComponent<StreamsTableProps> = ({
               <>
                 {getFormattedDate(created_at, t("ago"))}
                 <br />
-                {instance_type === InstanceType?.eval && (
+                {instance_type === InstanceType?.developer && (
```

Comment:
I think this should handle both `developer` and `eval` instances until all existing eval instances expires. 
When the deployment happens, `eval` instances won't be created but existing eval instances will continue to be returned. If this handles only `developer`, then for existing `eval` instances that have not expired yet there won't be any message about their expiration time.

@ajaypratap003 @wtrocki @riccardo-forina

Reply:
@machi1990 Thanks for letting us know about it. We will add both eval and developer .